### PR TITLE
Add neutral color scale documentation

### PR DESCRIPTION
**Pull request by langermank**
neutral gray scales for both light and dark themes

Adds a new section to the base scale page explaining how our neutral scales work. Check it out: https://primer-9c89ac4e24-26441320.drafts.github.io/foundations/color/base-scales

### CODE REVIEW

File: content/foundations/color/base-scales.mdx [hunk 1]
```diff
@@ -3,6 +3,71 @@ title: Base scales
...
+
+<img
+  width="960"
+  alt="Two neutral scales stacked on top of each other labelled with background, border, and text to show how the scales are used for different UI elements. In this graphic the text portion is highlighted."
+  src="https://github.com/user-attachments/assets/5cd87021-ce7e-4982-b081-95252146ccc9"
+/>
+
+### Adjusting contrast
+
+Contrast levels can be adjusted by moving up or down the scale. For example, the light high contrast theme utilizes steps one through 6 for backgrounds, 9 for borders, and 10 to black for text.
```
Comment (langermank) [inline content/foundations/color/base-scales.mdx:62]:
```suggestion
Contrast levels can be adjusted by moving up or down the scale. For example, the light high contrast theme utilizes steps 1 through 6 for backgrounds, 9 for borders, and 10 to black for text.
```

File: content/foundations/color/base-scales.mdx [hunk 2]
```diff
@@ -3,6 +3,71 @@ title: Base scales
 description:
   Primer base color scales makeup the foundation for all UI color considerations.
 ---
+## Neutral color scales
+
+<img
+  width="960"
+  alt="Two neutral scales stacked on top of each other labeled from 0 to 11. The top scale is the light scale, which starts with white and ends with black. The bottom scale is the dark scale, which starts with black and ends with white."
```
Comment (ericwbailey) [inline content/foundations/color/base-scales.mdx:10]: 💜

File: content/foundations/color/base-scales.mdx [hunk 3]
```diff
@@ -3,6 +3,71 @@ title: Base scales
...
+  src="https://github.com/user-attachments/assets/3081e29d-6581-401b-91da-c014eee9cf66"
+/>
+
+Primer's neutral scales offer white, black, and shades of gray between 0 and 11. There are two versions of the neutral scale: light, and dark. The light and dark scales are inverted respectively, with the light scale starting with white and the dark scale starting with black. By inverting the scales, light and dark themes are able to share many of the same functional color tokens without custom overrides.
```
Comment (ericwbailey) [inline content/foundations/color/base-scales.mdx:14]:
```suggestion
Primer's neutral scales offer white, black, and shades of gray between 0 and 11. There are two versions of the neutral scale: light, and dark. 

The light and dark scales are inverted respectively, with the light scale starting with white and the dark scale starting with black. By inverting the scales, light and dark themes are able to share many of the same functional color tokens without custom overrides.
```

This is a small nit, but I'm thinking we separate this into two separate paragraphs to both aid in reading ease, but also reinforce that A) there are two versions of the neutral scale, and then B) the two scales allow for increased color value sharing.

File: content/foundations/color/base-scales.mdx [hunk 4]
```diff
@@ -3,6 +3,71 @@ title: Base scales
...
+
+<img
+  width="960"
+  alt="A split view of how muted text colors reference the same base scale color in both light and dark themes."
+  src="https://github.com/user-attachments/assets/160e6384-3c33-4729-a6cf-96adce84f127"
+/>
+
+### Background
+
+The first five steps of the neutral scales are typically used for background colors. The two most commonly used background colors are `bgColor-default` and `bgColor-muted`. All contrast values for text and borders are calculated against `bgColor-muted` to ensure proper contrast ratios for both `muted` and `default`.
```
Comment (ericwbailey) [inline content/foundations/color/base-scales.mdx:24]: 💪

**Review by ericwbailey — APPROVED** [review]
Love this! One small optional nit, but it's great to see documentation on the underlying uniformity this logic sets up.

File: content/foundations/color/base-scales.mdx [hunk 5]
```diff
@@ -3,6 +3,73 @@ title: Base scales
...
```
Comment (mperrotti) [inline content/foundations/color/base-scales.mdx:26]: Technically it's the first 6 (if you don't count white), but I knew what you meant.

File: content/foundations/color/base-scales.mdx [hunk 6]
```diff
@@ -3,6 +3,73 @@ title: Base scales
...
+
+<img
+  width="960"
+  alt="Two neutral scales stacked on top of each other labelled with background, border, and text to show how the scales are used for different UI elements. In this graphic the background portion is highlighted."
+  src="https://github.com/user-attachments/assets/1a87a17d-ce23-4c1d-8fa0-ae7b2d70aaf2"
```
Comment (mperrotti) [inline content/foundations/color/base-scales.mdx:31]: When I first looked at these graphics the purple scrim threw me off. I thought maybe that was the section I was supposed to be focusing on, or it was a demo of what these colors look like near the light purple. Maybe switch to white with more opacity, use a background blur, or just bump up the opacity of the purple?

This is definitely not a blocker and is probably just my personal misunderstanding when looking at it for the first time before reading the text and giving it some thought.

**Review by mperrotti — APPROVED** [review]
This is very good documentation that shows off how smart and well-researched our neutral scale is. 🎉

**Comment by mperrotti** [comment]
Is there a similar PR coming to update the neutral scales in the Primitives Storybook too?

File: content/foundations/color/base-scales.mdx (hunk 5 above)
Comment (langermank) [reply content/foundations/color/base-scales.mdx:26]: I had a shockingly hard time with numbers on this PR 😂

File: content/foundations/color/base-scales.mdx (hunk 6 above)
Comment (langermank) [reply content/foundations/color/base-scales.mdx:31]: Thank you! I'm not in love with it either. I'll keep playing 👀

**Pull request merged**